index.html

```diff
@@ -0,0 +1,156 @@
+        </div>
+      </article>
+    </div>
+    <div class="section_two">
+      <article class="afloat">
+        <h2>Staying afloat</h2>
+        <p>Lobsterman John Aldridge fell into the shark-infested water of Long Island while his crewmates were asleep. He was kept afloat for more than 12 hours by his rubber boots, which he used as flotation devices. The coast guard rescued him 43 miles south of Montauk where he fell off his vessel.</p>
+        <figure>
+         <img src="img/boots.png" alt="Rubber Boots">
+        </figure>
+      </article>
+      <article class="bullets">
+        <div class="bullet_list">
+          <h2>LIST STYLE: BULLETS.</h2>
+          <h3>A ROUND-UP OF POPULAR AMMUNITION</h3>
+        </div>
+        <div>
+          <figure>
+            <img class="bullet_image" src="img/bullets.png" alt="Bullets">
+          </figure>
+          <ul>
+            <li class="padding_adjust_one"><span class="caliber_size_one">.223</span><br> <span class="manufacturer_name_one">REMINGTON</span> <span class="gunpowder_weight_one"><br>69gr BTHP</span></li>
+            <li class="padding_adjust_one"><span class="caliber_size_one">.308</span><br> <span class="manufacturer_name_one">WINCHESTER</span> <span class="gunpowder_weight_one">168gr BTHP</span></li>
+            <li class="padding_adjust_two"><span class="caliber_size_two">7.62x39mm</span> <span class="manufacturer_name_two">RUSSIAN</span><br><span class="gunpowder_weight_two">154gr JSP</span></li>
+            <li class="padding_adjust_three"><span class="caliber_size_three">7.62</span><br> <span class="manufacturer_name_two">THUMPER</span><br><span class="gunpowder_weight_two">220gr RN</span></li>
+            <li class="padding_adjust_three"><span class="caliber_size_three">7.62</span><br> <span class="manufacturer_name_two">THUMPER</span><br><span class="gunpowder_weight_one">240gr BTHP</span></li>
+          </ul>
+        </div>
+      </article>
+    </div>
+    <article class="abbey">
+      <h2>DISPLAYING (FAMOUS) BLOCKS</h2>
+      <figure>
+        <img class="beatles" src="img/abbey_road.png" alt="The Beatles crossing Abbey Road" >
+        <a href=""><img src="img/map.png" alt="Map of Abbey Road Location" class="map"></a>
+        <figcaption>Abbey Road is a thoroughfare located in the borough of Camden and the City of Westminster in London, runnning roughly northwest to southeast through St. John's Wood, near Lord's Cricket Ground. It is part of the B507. This road is best known for the Abbey Road Studios and the 1969 album, Abbey Road, by The Beatles.
+        <p class="credit">Photo: goo.gl/h44dc9</p>
```

Comment:
Using like this, it would make better sense to use `<small>`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small

> In HTML5, this element is repurposed to represent side-comments and small print, including copyright and legal text, independent of its styled presentation.